.github/workflows/trivy.yml

```diff
@@ -72,7 +52,7 @@ jobs:
         if: always()
         uses: aquasecurity/trivy-action@master
         with:
-          image-ref: "ghcr.io/catenax-ng/tx-sd-factory/sdfactory:latest"
+          image-ref: "ghcr.io/cofinity-x-sd-factory-core/sdfactory:latest"
```

Comment:
Security scans also need to stay checking the images build by this open source project and must not check images for commercial providers